CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 All notable changes to this project will be documented in this file. This project adheres to Semantic Versioning.
 
+### 3.0.1
```

Comment:
Should be 3.1.0 since it's not a bugfix, but a feature

Reply:
Ok 👍

Reply:
Changed